### ISSUE TITLE
Framing Table not craftable

### ISSUE THREAD

**Issue by KingHorse2015** [scalda/FoolCraft#298]
- [X] I am running the latest version of FoolCraft
- [X] I am running FoolCraft via Curse/Twitch Launcher
- [X] I can reproduce this issue consistently
     - [ ] In single player
     - [X] In multiplayer
- [ ] I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I'm having the same problem.
- [ ] I am crashing and can provide my crash report(s)

## SUGGESTION/HELP
- [X] I would like to propose an additional mod, config change, custom recipe, etc.
- [ ] I am requesting help with installing or running the pack

Can not craft a Framing table, get a helmet instead.

Put 5 woodtrims in a head shape for framing table, you get something else

![2017-04-18_19 56 42](https://cloud.githubusercontent.com/assets/15171668/25145349/3bb550f4-2471-11e7-96de-93dd7428f6bc.png)
![2017-04-18_19 56 33](https://cloud.githubusercontent.com/assets/15171668/25145348/3bb02ed0-2471-11e7-9781-af76efdae0be.png)

**Comment by Jo-Mamma75** [comment]
Duplicate of multiple other issues. #107 #125 #209 

Apparently the part in the template that stated "I have searched for this issue previously and it was either (1) not previously reported, or (2) previously fixed and I'm having the same problem" was too difficult to understand.

I don't mean to sound salty about it but we get a lot of issues reported here that are not necessary because people don't look at other issues or even use the search bar at the top.

Closing issue as duplicate.
